sql-statements/sql-statement-admin-cancel-ddl.md

```diff
@@ -0,0 +1,46 @@
+---
+title: ADMIN CANCEL DDL
+summary: TiDB 数据库中 ADMIN CANCEL DDL 的使用概况。
+---
+
+# ADMIN CANCEL DDL
+
+`ADMIN CANCEL DDL` 语句用于取消当前正在运行的 DDL 作业。可以通过 `ADMIN SHOW DDL JOBS` 语句获取 DDL 作业的 `job_id`。
+
+## 语法图
+
+**AdminStmt:**
+
+![AdminStmt](/media/sqlgram/AdminStmt.png)
+
+**NumList:**
+
+![NumList](/media/sqlgram/NumList.png)
+
+## 示例
+
+可以通过 `ADMIN CANCEL DDL JOBS` 语句取消当前正在运行的 DDL 作业，并返回对应作业是否取消成功：
+
+{{< copyable "sql" >}}
+
+```sql
+ADMIN CANCEL DDL JOBS job_id [, job_id] ...;
+```
+
+如果取消失败，会显示失败的具体原因。
+
+> **注意：**
+>
+> + 只有该操作可以取消 DDL 作业，其他所有的操作和环境变更（例如机器重启、集群重启）都不会取消 DDL 作业。
+>
+> + 该操作可以同时取消多个 DDL 作业，可以通过 `ADMIN SHOW DDL JOBS` 语句来获取 DDL 作业的 `job_id`。
+>
+> + 如果希望取消的作业已经完成，取消操作将失败。
+
+## MySQL 兼容性
+
+`ADMIN CANCEL DDL` 语句是 TiDB 对 MySQL 语法的扩展。
+
+## 另请参阅
+
+* [`ADMIN SHOW DDL [JOBS,QUERIES]`](/sql-statements/sql-statement-admin-show-ddl.md)
```

Comment:
```suggestion
* [`ADMIN SHOW DDL [JOBS|QUERIES]`](/sql-statements/sql-statement-admin-show-ddl.md)
```

Reply:
@Joyinqin Please also confirm and update changes in these two comments by @djshow832 to the English version. Thanks!

Reply:
I updated it in https://github.com/pingcap/docs/pull/4184, and thx for the comment!